rules/naming/src/VariableRenamer.php

```diff
@@ -4,6 +4,7 @@
 
 namespace Rector\Naming;
 
+use Closure;
```

Comment:
this is false positive as there is already `use PhpParser\Node\Expr\Closure;` means `\Closure` is invalid.

Reply:
fixed.